src/FSharp.Formatting.ApiDocs/GenerateModel.fs

```diff
@@ -2261,12 +2261,30 @@ module internal SymbolReader =
             let sum = el.Element(XName.Get "summary")
 
             match sum with
-            | null when String.IsNullOrEmpty el.Value -> cmds, ApiDocComment.Empty, None
+
+            // sum can be null with null/empty el.Value when an non-"<summary>" XML element appears
+            // as the only '///' documentation command:
+            //
+            // 1.
+            //  // Not triple-slash ccomment
+            //  /// <exclude/>
+            //
+            // 2.
+            //  /// <exclude/>
+            //
+            // So, we need to let the 'null' case handle this to extract the <exclude> if it's there
+            //
+            // | null when String.IsNullOrEmpty el.Value ->
+            //     cmds, ApiDocComment.Empty, None
+
             | null ->
                 // We let through XML comments without a summary tag. It's not clear
                 // why as all XML coming through here should be from F# .XML files
                 // and should have the tag.  It may be legacy of previously processing un-processed
                 // XML in raw F# source.
+                //
+                // 9-Jan-23: See comment above for at least one reason why we pass through here now
```

Comment:
This journal entry seems a tad excessive. You explained the reasoning not ten lines ago.

Reply:
Well, it's a single line pointing at the explanation, in response to the question raised in the previous developer's comment "why are we here". How do you want to improve this?

Reply:
I removed all the comments. I see your point (to an extent), however: the original comment either needs a response (in context), or just removing. My approach is not to remove comments, but respond to them in a timeline to help out developers down the line.  I think however nothing is lost - if you think it's excessive, others may do too.